src/main/java/crazypants/enderio/machine/obelisk/inhibitor/TileInhibitorObelisk.java

```diff
@@ -38,7 +40,7 @@ protected boolean processTasks(boolean redstoneCheck) {
 
   @Override
   public float getRange() {
-    return AVERSION_RANGE.getFloat(getCapacitorData()) / 2;
+    return (float) AbstractBlockObelisk.DUMMY;
```

Comment:
if you delete the old code instead of commenting it out, it will be a pain to reinstate it later once the keys are in again